iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/BusAdapter.java

```diff
@@ -188,22 +189,28 @@ public int getConnectedTerminalCount() {
 
     @Override
     public Iterable<Line> getLines() {
-        throw MergingView.createNotImplementedException();
+        return getLineStream().collect(Collectors.toList());
```

Comment:
I think it's not efficient to implement thinks like this: to create the Iterable, you will iterate over all the elements (collect), and then the client of the API will also do it, so the list is read twice.

I think it would be better to use Iterables.concat(). There is exactly the same issue in the MergingViewIndex... we should fix